tests/integration/conftest.py

```diff
@@ -89,9 +89,10 @@ def assert_cairo_test_cases(
 
 @pytest.fixture(name="devnet_gateway_url", scope="session")
 def devnet_gateway_url_fixture(devnet_port: int):
-    proc = run_devnet(["poetry", "run", "starknet-devnet"], devnet_port)
-    yield f"http://localhost:{devnet_port}"
-    proc.kill()
+    with base_protostar_toml():
```

Comment:
why not using here the fixture you introduced in this PR?

Reply:
removed usage